Week-1/assignment_1.2/index.html

```diff
@@ -0,0 +1,107 @@
+    <link rel="stylesheet"
+        href="https://maxst.icons8.com/vue-static/landings/line-awesome/line-awesome/1.3.0/css/line-awesome.min.css" />
+    <link rel="stylesheet" href="./style.css" />
+    <title>Document</title>
+</head>
+
+<body>
+    <!-- sidebar -->
+    <aside>
+        <i class="las la-bars"></i>
+        <i class="las la-home"></i>
+        <i class="las la-search"></i>
+        <i class="las la-volume-up"></i>
+        <i class="las la-user"></i>
+        <i class="las la-cog"></i>
+        <i class="lab la-spotify"></i>
+        <i class="lab la-soundcloud"></i>
+        <!-- Side bar ends -->
+    </aside>
+    <main>
+        <header>
+            <div class="album-img-wrapper">
+                <img src="https://vif1g.csb.app/src/assets/cattyboard.jpg" alt="album-img" />
+            </div>
+            <div class="album-details">
+                <div class="album-content">
+                    <h1>CattyBoard Top 100 Single Charts(11.06.36)</h1>
+                    <h3>Unknown Artist</h3>
+                    <h3>2016.Charts.100 songs</h3>
+                </div>
+                <div class="album-action-btn">
+                    <span>
+                        <i class="las la-play"></i>
+                        <p>Play all</p>
+                    </span>
+                    <span>
+                        <i class="las la-plus"></i>
+                        <p>Add to</p>
+                    </span>
+                    <span>
+                        <i class="las la-ellipsis-h"></i>
+                        <p>More</p>
+                    </span>
+                </div>
+            </div>
+        </header>
+        <div>
+            <table>
+                <tr>
+                    <td>1.One Dance</td>
+                    <td>Crake feat CatKid & Cyla</td>
+                    <td>2:54</td>
+                    <td><a href="#">catty cloud sync</a></td>
+                </tr>
+                <tr>
+                    <td>2.Panda</td>
+                    <td>Cattee</td>
+                    <td>4:06</td>
+                    <td><a href="#">catty cloud sync</a></td>
+                </tr>
+                <tr>
+                    <td>3.Can't Stop the Feeling!</td>
+                    <td>Catin Cimberlake</td>
+                    <td>3:56</td>
+                    <td><a href="#">catty cloud sync</a></td>
+                </tr>
+                <tr>
+                    <td>4.Work From Home</td>
+                    <td>Cat Harmony feat Colla</td>
+                    <td>3:34</td>
+                    <td><a href="#">catty cloud sync</a></td>
+                </tr>
+            </table>
+        </div>
+        <p></p>
+    </main>
+    <!-- navbar bottom -->
+    <bottom-nav>
+        <div class="action-bar-left">
+            <div class="action-bar-img-wrapper">
+                <img class="action-bar-img" src="https://vif1g.csb.app/src/assets/cattyboard.jpg" alt="" />
```

Comment:
keep habit to add content in alt attribute

Reply:
Creating a commit right away.